pipelines/rh-push-to-registry-redhat-io/rh-push-to-registry-redhat-io.yaml

```diff
@@ -295,6 +295,8 @@ spec:
       workspaces:
         - name: data
           workspace: release-workspace
+      runAfter:
+        - apply-mapping
```

Comment:
Fine with this, but maybe it should run after `push-snapshot`? Do (or even can) we want to sign something before it is pushed to quay?

Reply:
I dropped this from this PR. Another PR should be opened that addresses all pipelines